rust-libretro/src/environment.rs

```diff
@@ -1921,18 +2023,23 @@ pub unsafe fn set_core_options_update_display_callback(
 /// implementation may therefore pass `NULL` in order
 /// to test whether the callback is supported.
 #[proc::context(GenericContext)]
-pub unsafe fn set_variable(callback: retro_environment_t, value: retro_variable) -> bool {
+pub unsafe fn set_variable(
```

Comment:
`RETRO_ENVIRONMENT_SET_VARIABLE` accepts an input of `NULL` to test for support.